Simplified restart and shutdown commands.

sid-kap opened the pull request.

sid-kap committed:
Simplified restart and shutdown commands.

samm81 commented:
Huh, I didn't know about those. They actually make these commands very silly and redundant, go ahead and just delete the files and I'll merge the PR.

sid-kap committed:
Removed restart and shutdown scripts.
There are already one-word commands for these purposes (/sbin/reboot and /sbin/restart) so there's no need for an alias that does the same thing.

sid-kap commented:
Done.
